### PR TITLE
[CDAP-4986] UI Complex Schema Integration Part 2

### PR DESCRIPTION
**Pull request by elfenheart**
Feature parity should be achieved with this PR.

Related: https://github.com/caskdata/hydrator-plugins/pull/310

### CODE REVIEW

**Comment by ajainarayanan** [comment]
Noticed two things,
- We might want to add infinite scroll back, on importing huge schemas, opening node configs got delayed.
- On import on empty schema in stream (with just ts & headers) has a JS error.

Other than that it looks awesome! 👍

**Comment by elfenheart** [comment]
will add infiinite scroll in a separate PR

File: cdap-ui/app/directives/complex-schema/complex-schema.js [hunk 1]
```diff
@@ -178,10 +188,11 @@ angular.module(PKG.name+'.commons')
       model: '=ngModel',
       recordName: '=',
       parentFormatOutput: '&',
-      isDisabled: '='
+      isDisabled: '=',
+      schemaPrefix: '='
     },
     link: (scope, element) => {
-      $compile('<my-complex-schema ng-model="model" record-name="recordName" hide-header="true" parent-format-output="parentFormatOutput()" is-disabled="isDisabled"></my-complex-schema')(scope, (cloned) => {
+      $compile('<my-complex-schema ng-model="model" record-name="recordName" hide-header="true" parent-format-output="parentFormatOutput()" is-disabled="isDisabled" schema-prefix="schemaPrefix"></my-complex-schema')(scope, (cloned) => {
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/complex-schema/complex-schema.js]: Can we try es6 string literal? Then we could have all the formatting. Right now it just expands as one single long line in the editor.

**Comment by ajainarayanan** [comment]
Just a minor comment. 👍 LGTM. 🎱

**Comment by ajainarayanan** [comment]
👍 LGTM

**Comment by elfenheart** [comment]
Squashed and demoed to @bdmogal

**Pull request merged**